src/components/ec-text-field/ec-text-field.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="ec-text-field">
+    <div
```

Comment:
1. There is a `<label>` element.
2. We need to add basic accessibility: when I click on label, the input must gain a focus. This can be achieved by adding `<label for="input-id">Label</label><input id="input-id">` (the ID must be unique, thus generated) or just simply `<label>Label<input></label>` which doesn't require any ID.